embark tests are slooooow

### Bug Report

#### Summary

<!-- Provide a summary describing the problem you are experiencing. -->

Embark tests are extremely slow. They are up to 10x slower than using mocha & ganache-cli independently of embark.

Here is a screenshot of running tests before converting to embark (using https://github.com/Giveth/liquidpledging commit 41a087d5093746e87e5031f3f93c0c86db7f347d with command `npm run test test/NormalOperation.js`):

<img width="759" alt="screen shot 2018-11-15 at 12 26 11 pm" src="https://user-images.githubusercontent.com/30963004/48580694-be680900-e8d4-11e8-8845-d3d627753e99.png">

And here is a screenshot of running tests using the embark framework. Note the tests are essentially the same, the only difference is I'm using embark to start the simulator & execute the tests. I'm still using the same calls & js wrappers as the above screenshot. (using https://github.com/Giveth/liquidpledging/tree/embark commit 9376cfb1e296e67d66d6f6d665f560ecfb3e427e with command `embark test test/NormalOperation.js`)

<img width="864" alt="screen shot 2018-11-15 at 12 34 46 pm" src="https://user-images.githubusercontent.com/30963004/48580824-0d15a300-e8d5-11e8-81ea-553b39fe2ba2.png">


Embark does include the gas reporter. If that is the cause of the slowdown, I would recommend not running the gas reporter by default and enabling it with a flag

#### Please provide additional information about your system

**OS**: macOS mojave
**Embark Version**: master commit 21dff441d1440a2cac6eb791e3c921da248f741d
**Node Version**:
**NPM Version**:

#### Sometimes issues are related to Embark's installation. Can you provide information on how Embark was installed?

@perissology , I can't manage to run your tests (with `npm run test test/NormalOperation.js`). I went back to master and I get the following error. Do you think it's related to Windows?

```
"C:\Program Files\JetBrains\WebStorm 2018.1.1\bin\runnerw.exe" "C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js" test --scripts-prepend-node-path=auto test/NormalOperation.js

> giveth-liquidpledging@1.0.1 test C:\dev\userProjects\liquidpledging
> npm run build && mocha --harmony "test/NormalOperation.js"


> giveth-liquidpledging@1.0.1 build C:\dev\userProjects\liquidpledging
> npm run compile


> giveth-liquidpledging@1.0.1 compile C:\dev\userProjects\liquidpledging
> npm run compile:sol &&  npm run compile:js


> giveth-liquidpledging@1.0.1 compile:sol C:\dev\userProjects\liquidpledging
> solcpiler --solc-version v0.4.18+commit.9cf6e910 -i './contracts/**/*.sol' ./node_modules/@aragon/os/contracts/{kernel/Kernel.sol,acl/ACL.sol,apps/AppProxyUpgradeable.sol} --output-sol-dir build/contracts

Error: Missing source for import "./IKernel.sol";
Looked in dir: ./node_modules/@aragon/os/contracts/kernel\node_modules\@aragon\os\contracts\kernel\IKernel.sol, contracts: C:\dev\userProjects\liquidpledging\contracts\node_modules\@aragon\os\contracts\kernel\node_modules\@aragon\os\contracts\kernel\IKernel.sol, src: C:\dev\userProjects\liquidpledging\src\node_modules\@aragon\os\contracts\kernel\node_modules\@aragon\os\contracts\kernel\IKernel.sol, npm: C:\dev\userProjects\liquidpledging\node_modules\node_modules\@aragon\os\contracts\kernel\node_modules\@aragon\os\contracts\kernel\IKernel.sol, and libs
    at matches.forEach (C:\dev\userProjects\liquidpledging\node_modules\solcpiler\js\solcpiler.js:362:30)
    at Array.forEach (<anonymous>)
    at Solcpiler.resolveImportsFromFile (C:\dev\userProjects\liquidpledging\node_modules\solcpiler\js\solcpiler.js:347:13)
    at Object.keys.forEach (C:\dev\userProjects\liquidpledging\node_modules\solcpiler\js\solcpiler.js:238:12)
    at Array.forEach (<anonymous>)
    at Solcpiler.generateStandardJson (C:\dev\userProjects\liquidpledging\node_modules\solcpiler\js\solcpiler.js:193:31)
    at Promise.all.then (C:\dev\userProjects\liquidpledging\node_modules\solcpiler\js\solcpiler.js:81:14)
    at <anonymous>

> giveth-liquidpledging@1.0.1 compile:js C:\dev\userProjects\liquidpledging
> babel -d lib/ js/

js\contracts.js -> lib\contracts.js
js\liquidPledgingState.js -> lib\liquidPledgingState.js
Error: Cannot find module '../build/Kernel.json'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\dev\userProjects\liquidpledging\lib\contracts.js:8:22)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\dev\userProjects\liquidpledging\index.js:3:19)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\dev\userProjects\liquidpledging\test\NormalOperation.js:6:72)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at C:\dev\userProjects\liquidpledging\node_modules\mocha\lib\mocha.js:231:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\dev\userProjects\liquidpledging\node_modules\mocha\lib\mocha.js:228:14)
    at Mocha.run (C:\dev\userProjects\liquidpledging\node_modules\mocha\lib\mocha.js:514:10)
    at Object.<anonymous> (C:\dev\userProjects\liquidpledging\node_modules\mocha\bin\_mocha:480:18)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
npm ERR! Test failed.  See above for more details.

Process finished with exit code 1
```

--- a/src/cmd/cmd_controller.js
+++ b/src/cmd/cmd_controller.js
@@ -642,7 +642,9 @@ class EmbarkController {
         });
         engine.startService("codeGenerator");
         engine.startService("codeRunner");
-        engine.startService("codeCoverage");
+        if (options.coverage) {
+          engine.startService("codeCoverage");
+        }
         engine.startService("testRunner");
         callback();
       },

